build/yaml/templates/nuget-versioning-steps.yml

```diff
@@ -53,7 +53,7 @@ steps:
 
     # Configure version suffix based on deployment ring
     if ($deploymentRing.ToLowerInvariant() -ne "stable") {
-      $nugetVersionSuffix = $deploymentRing + $vs;
+      $nugetVersionSuffix = $deploymentRing;
```

Comment:
I'll need to check what to do with this one, but I can take care of that separately.